### PR TITLE
docs(README): update README

### PR DESCRIPTION
**Pull request by ghost**
## Description

- An initial revision of the autoware README
- Edited the intro text to remove information that was copy/pasted from the AWF profile page and to make it Autoware specific
- added the PRs welcome badge and adjusted the size and position of the Discord badge to match
- edited the Documentation section slightly
- added a note in the Contributing section about asking technical questions in the Q&A category
- added Support guidelines to the useful resources list

Signed-off-by: Lalith Vipulananthan <lalith.vipulananthan@tier4.jp>

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +36,14 @@ See the [documentation site](https://autowarefoundation.github.io/autoware-docum
...
   - Make sure to follow the [Contribution Guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
   - Take a look at Autoware's [various working groups](https://github.com/autowarefoundation/autoware-projects/wiki#working-group-list) to gain an understanding of any work in progress and to see how projects are managed.
-- If you have any questions or ideas, feel free to start a discussion on [GitHub Discussions in autowarefoundation/autoware](https://github.com/autowarefoundation/autoware/discussions).
+- If you have any technical questions, you can start a discussion in the [Q&A category](https://github.com/autowarefoundation/autoware/discussions/categories/q-a) to request help and confirm if a potential issue is a bug or not.
 
-### Useful resources
+## Useful resources
 
 - [Autoware Foundation homepage](https://www.autoware.org/)
+- [Support guidelines](https://autowarefoundation.github.io/autoware-documentation/main/help/support-guidelines/).
```
Comment (yukkysaito) [inline README.md]:
```suggestion
- [Support guidelines](https://autowarefoundation.github.io/autoware-documentation/main/help/support-guidelines/)
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+Autoware is an open-source software stack for self-driving vehicles, built on the [Robot Operating System (ROS)](https://www.ros.org/). It includes all of the necessary functions to drive an autonomous vehicles from localization and object detection to route planning and control, and was created with the aim of enabling as many individuals and organizations as possible to contribute to open innovations in autonomous driving technology.
 
-[![Discord](https://img.shields.io/discord/953808765935816715?label=Join%20Autoware%20Discord&style=for-the-badge)](https://discord.gg/Q94UsPvReQ)
+![Autoware architecture](https://static.wixstatic.com/media/984e93_552e338be28543c7949717053cc3f11f~mv2.png/v1/crop/x_0,y_1,w_1500,h_879/fill/w_863,h_506,al_c,usm_0.66_1.00_0.01,enc_auto/Autoware-GFX_edited.png)
 
-### Documentation
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://makeapullrequest.com)
```
Comment (yukkysaito) [inline README.md]: I think this is bette rto a link to the contribution guide in the autoware document.
Users may be confused if the autoware document's contribution guide does not match the description in the link.

Comment (ghost) [reply README.md]: The badge links to a free online training course explaining GitHub basics which is also something that contributors may need to know, but I see your point that the first reference should be the contribution guidelines in the documentation.

I will remove the badge and propose adding a link to that training course in the documentation instead.

Comment (yukkysaito) [reply README.md]: Ok. I'm not picky about it.:+1: 
@kenji-miyake What do you think?

Comment (kenji-miyake) [reply README.md]: > I will remove the badge and propose adding a link to that training course in the documentation instead.

@yukkysaito I think this is good!

**Review by xmfcx — APPROVED** [review]
Looks great, thanks!

**Pull request merged**